src/Components/PetInformation.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
```

Comment:
Love the decision to put all this in its own component. Keeps Pet.js very clean with the conditional render of one or the other component.